### DISCUSSION TITLE
[ FF7 Steam ] FFNx does not run when game installed on another HDD path

### DISCUSSION THREAD

**Discussion by wonx** [julianxhokaxhiu/FFNx#196] [General]
Hi,

I know it's not supported, but I'm having trouble installing FFNx alongside 7th heaven 2.0. 

The game (from steam) is installed at `D:\SteamLibrary\steamapps\common\FINAL FANTASY VII`, and 7th Heaven 2.0 at `C:\Program Files (x86)\7th Heaven`. I am following these instructions from this website, using a fresh install of FF7 from Steam and 7th heaven 2.0 (no mods used yet):

> How to install on 7h 2.x
> WARNING: This method is NOT SUPPORTED officially by 7h team. Please use at your own risk. DO NOT approach 7h team if something is not working properly.
> 
> 1. Go to your 7h installation path
> 2. Go to Resources\Game Driver
> 3. Rename 7H_GameDriver.dll to _7H_GameDriver.dll
> 4. Install FFNx following instructions here: https://github.com/julianxhokaxhiu/FFNx#1998-eidos-release

>> (from these instructions):
> 4.1 Install the game on this path: C:\Games\Final Fantasy VII
> 4.2 Update your game to v1.02 ( https://www.gamefront.com/games/final-fantasy-7-advent-children/file/final-fantasy-7-retail-v1-02-patch )
> 4.3 Download the latest FFNx-FF7_1998 release here: https://github.com/julianxhokaxhiu/FFNx/releases
> 4.4 Extract the ZIP content next to ff7.exe file
> 4.5 Double click on FFNx.reg
> 4.6 Click on Yes.
> Enjoy!

> 5. Rename FFNx.dll to 7H_GameDriver.dll
> 6. Click Play and enjoy!

Does that mean that I also need to install the v1.02 patch? Do I have to rename FF7.exe from that patch to ff7_en.exe, or both need to be in the game directory? Should 7th heaven then point to FF7.exe or ff7_en.exe?

Regarding point 5. That file is in the game dir, not in 7th heaven's dir, right?

In any case, I can't seem to launch FFVII using Vulkan...

**Comment by julianxhokaxhiu** [comment]
Hi,

you're mix and matching many issues at once.

1) It would be great if you intend to use 7h 2.x to run the game at least once using 7h 2.x natively.
2) If that works fine, then you can use instructions to install FFNx on top of 7h. Just follow those, no renaming, no patch to be install. If that would have been required it would have been mentioned in the steps :)
3) Vulkan is a technical preview which is not meant to be used daily. If you read the cfg file, you'll see this mentioned as well.

Hear you on Discord.

Cheers!

**Comment by wonx** [comment]
Ok, I discussed the issue on Discord, but I am not even able to use FFNx with a fresh install of FF7 from Steam (without 7th heaven or anything). As soon as I start the game, the launcher opens, I click on "Play", and then the game goes fullscreen for a couple of seconds and crashes, going back to the launcher.

**Comment by julianxhokaxhiu** [comment]
Post your `FFNx.log` file in here. That's weird. I think you're confusing the 7h `ff7.exe` with Steam `ff7_en.exe` which uses two different way of installing the driver.

**Comment by wonx** [comment]
We already did some tests on the discord channel, but these are the contents of FFNx.log, using a vanilla Steam install and FFNx-Steam-v1.6.1.0.zip:

```
[00000000] INFO: FFNx driver version 1.6.1.0
[00000000] TRACE: v1: 0x99CE0805, v2: 0xAC0D8B00
[00000000] INFO: Auto-detected version: FF7 1.02 US English
[00000000] TRACE: Detected Steam edition.
```

**Comment by julianxhokaxhiu** [comment]
This issue can't be replicated as another user is having FF7 installed on another HDD too and works.

Closing as invalid.

**Comment by damur82** [comment]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/14207956/FFNx.log)
hi i have a game crush and i domt know if its be auss of having incompatible mods or what

**Reply by julianxhokaxhiu** [reply]
Feel free to use Discord support channels. Thanks.
